### DISCUSSION TITLE
ConvertDBForHistoricalConfigurations.exe Location?

### DISCUSSION THREAD

**Discussion by mpolicastro** [udotdevelopment/ATSPM#196] [Q&A] [answered]
Where can I find the ConvertDBForHistoricalConfigurations.exe file? It does not appear to be in the 4.3 Version zip folder. The installation manual says it is in the installation media but I'm new to updating an ATSPM system and don't know where that is. Any help would be appreciated.

**Comment by avetraffic** [comment, marked as answer]
We're releasing a patch that includes the ConvertDBForHistoricalConfigurations.exe file in the next few weeks. Or you can extract it from the source files as noted by others in #157.

**Comment by mpolicastro** [comment]
From #157: ian-cameron has extracted and edited the connection string in the .config file. However, #157 does not provide more information or an explanation about how to do this. Are there instructions for this process anywhere?

**Reply by avetraffic** [reply]
There currently are no instructions on how to update it without the ConvertDBForHistoricalConfigurations.exe file. You would have to download the project source code which is located in each release alongside the executables, open the ConvertDBForHistoricalConfigurations project in something like Visual Studio, and run it from there after editing the connection string to match your setup. But we should hopefully have the patch out by the end of next week that should make this unncessary.

**Reply by avetraffic** [reply]
We've released 4.3.1 which includes the ConvertDBForHistoricalConfigurations executable. The Installation Manual has also been updated.
